sycl/include/sycl/ext/intel/experimental/fpga_annotated_properties.hpp

```diff
@@ -315,44 +315,6 @@ struct PropertyMetaInfo<read_write_mode_key::value_t<Mode>> {
 
 } // namespace detail
 
-// 'buffer_location' and mmhost properties are pointers-only
-template <typename T, int N>
-struct is_valid_property<T, buffer_location_key::value_t<N>>
```

Comment:
@broxigarchen  "is_valid_property" is something we created or is that part of the standard boiler-plate for all oneapi properties? If we created it, I do not mind removing it.

Reply:
Yup